draft-ietf-webpush-protocol.xml

```diff
@@ -799,6 +925,16 @@ HEADERS      [stream 4] +END_STREAM
           superceded (e.g. the current number of unread emails).
         </t>
       </section>
+      <section anchor="sets-streams" title="Subscription Sets and Concurrent HTTP/2 streams">
+        <t>
+          If the push service requires that the user agent use push message subscription sets,
+          then it SHOULD limit the number of concurrently active streams with the
+          SETTINGS_MAX_CONCURRENT_STREAMS parameter within a HTTP/2 SETTINGS frame <xref target="RFC7540"/>. 
+          
+          The user agent SHOULD be limited to one concurrent stream to manage push message
+          subscriptions and one concurrent stream per subscription set returned by the push service.

```

Comment:
Changing to MAY.  Regarding "this choice limits concurrency", I thought that was implicit in the text. Did you want to call out that this may force the UA to serialize subscription requests? Of course, if it had 2 streams (subscriptions+set messages) then it could use both in a pinch for subscription management.